cogs/triggered/triggered.py

```diff
@@ -0,0 +1,91 @@
+"""Triggered cog
+`triggered from spoopy.
+"""
+
+import os
+import urllib.request
+import discord
+from redbot.core import commands, data_manager, Config
+from PIL import Image, ImageChops, ImageOps
+
+SAVE_FOLDER = "data/lui-cogs/triggered/"  # Path to save folder.
+SAVE_FILE = "settings.json"
+AVATAR_URL = "https://cdn.discordapp.com/avatars/{0.id}/{0.avatar}.png?size=512"
+
+# def checkFolder():
+#     """Used to create the data folder at first startup"""
+#     if not os.path.exists(SAVE_FOLDER):
+#         print("Creating " + SAVE_FOLDER + " folder...")
+#         os.makedirs(SAVE_FOLDER)
+
```

Comment:
Should be safe to remove this code block since we're not using it in v3 anymore.